README.md

```diff
@@ -0,0 +1,87 @@
+# COSIMA Spack Configuration
+
+This repository contains the spack configuration and the spack environments used
+by COSIMA to deploy software on gadi.
+
+## Instalation instructions
```

Comment:
typo: Installation

Reply:
@aekiss you can also use the suggestion button when reviewing and then the changes can just implemented with a click from the PR owner.

https://www.youtube.com/watch?v=rCARUlE2C0M

Reply:
awesome, thanks @navidcy, I was just thinking that would be a good feature to have!

Reply:
@github thought it first ;)